src/ralph/dc_view/serializers/models_serializer.py

```diff
@@ -121,13 +125,14 @@ class Meta:
 class PDUSerializer(serializers.ModelSerializer):
     model = serializers.CharField(source='model.name')
     orientation = serializers.CharField(source='get_orientation_desc')
+    url = serializers.CharField(source='get_absolute_url')
 
     def get_type(self, obj):
         return TYPE_PDU
 
     class Meta:
         model = DataCenterAsset
-        fields = ('model', 'sn', 'orientation')
+        fields = ('model', 'sn', 'orientation', 'url')
 
 
 class RackSerializer(AdminLinkMixin, serializers.ModelSerializer):
```

Comment:
please separate this serializer into: ``RackBaseSerializer`` and ``RackCreateSerializer`` (which inherit from ``RackBaseSerializer``)